lib/_footer.erb

```diff
@@ -1,5 +1,8 @@
     <footer id="footer" class="text-center">
       Gluegun project is licensed under Apache 2.0 License.
     </footer>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
```

Comment:
I think we need to move these three lines to the script definitions on index.erb, as I get the error displayed below (and Bootstrap and jQuery are already loaded over there). However, when I moved these three lines to index.erb, and deleted the Bootstrap and jQuery definitions there, I was able to solve the problem.

![screen shot 2018-03-16 at 6 54 58 pm](https://user-images.githubusercontent.com/27314853/37550520-ed0d6e2e-294b-11e8-95d1-3cdab67ef2ef.png)

Reply:
@kaankabalak @deekoder There are 2 Bootstrap.js used. Removing the first which is called in _header.erb would fix this issue. Bootstrap 4 requires popper.js as a dependency module and popper should be called prior to bootstrap.

Reply:
oh sorry. I will remove it.

Reply:
@rushenn so how about i remove bootstrap completely and let you figure out which version to use in the default style?

Reply:
@deekoder Yeah that would be better. I'm planning to drop Bootstrap.